gradle.properties

```diff
@@ -4,3 +4,4 @@ quarkusPlatformArtifactId=quarkus-bom
 quarkusPluginId=io.quarkus
 quarkusPlatformGroupId=io.quarkus.platform
 quarkusPlatformVersion=2.12.0.Final
+org.gradle.jvmargs=-Xmx4096m
```

Comment:
Is this related to the removal of the dependency?

Reply:
oops, forgot to not commit this config. it's a local config, to avoid the no java heap space error that happens a lot when working locally with the ee extension